packages/@o3r/apis-manager/schematics/index.it.spec.ts

```diff
@@ -16,18 +17,27 @@ import * as path from 'node:path';
 
 describe('new otter application with apis-manager', () => {
   test('should add apis-manager to existing application', () => {
-    const { workspacePath, projectName, isInWorkspace, untouchedProjectPath, o3rVersion } = o3rEnvironment.testEnvironment;
+    const { workspacePath, appName, isInWorkspace, untouchedAppPath, libraryPath, untouchedLibPath, o3rVersion } = o3rEnvironment.testEnvironment;
     const execAppOptions = {...getDefaultExecSyncOptions(), cwd: workspacePath};
-    packageManagerExec({script: 'ng', args: ['add', `@o3r/apis-manager@${o3rVersion}`, '--skip-confirmation', '--project-name', projectName]}, execAppOptions);
+    const packageManager = getPackageManager();
+    const isYarnTest = packageManager.startsWith('yarn');
```

Comment:
could this 'isYarnTest' constant be exported by `o3rEnvironment.testEnvironment`